test/unit/redux/reducers/tideDashboardPatients.test.js

```diff
@@ -69,4 +69,23 @@ describe('tideDashboardPatients', () => {
       ] } });
     });
   });
+
+  describe('fetchPatientFromClinicSuccess', () => {
+    it('should set state to initial empty state', () => {
```

Comment:
Description didn't get updated from it's original copy :)

Reply:
Yep :)